src/ServiceControl.Config/Framework/Rx/RxPropertyChanged.cs

```diff
@@ -2,28 +2,22 @@
 {
     using System;
     using Caliburn.Micro;
-    using PropertyChanging;
     using ReactiveUI;
+    using PropertyChangingDoNotNotify = PropertyChanging.DoNotNotifyAttribute;
+    using PropertyChangedDoNotNotify = PropertyChanged.DoNotNotifyAttribute;
 
     public class RxPropertyChanged : ReactiveObject, INotifyPropertyChangedEx
     {
-        [DoNotNotify]
-        [PropertyChanged.DoNotNotify]
-        [Obsolete("Use SuppressChangeNotifications() instead.", true)]
+        [PropertyChangingDoNotNotify]
+        [PropertyChangedDoNotNotify]
```

Comment:
Was the removal of the `[Obsolete]` intentional?

Reply:
I couldn't see a reason why it is needed. This is not called by us nor deprecated by the hierarchy in any way

Reply:
@danielmarbach if it is not used maybe can be removed? if it is there, I think we should keep the [obsolete] to avoid people using it by mistake.